instaclustr/resource_azure_peering.go

```diff
@@ -0,0 +1,178 @@
+package instaclustr
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAzureVpcPeering() *schema.Resource {
+	return &schema.Resource{
+		Create: AzureresourceVpcPeeringCreate,
+		Read:   AzureresourceVpcPeeringRead,
+		Update: resourceVpcPeeringUpdate,
+		Delete: resourceVpcPeeringDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceVpcPeeringStateImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"cdc_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"peer_resource_group": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"peer_suscription_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"peer_vpc_net": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"resource_group": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"suscription_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"peer_subnet": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"peer_subnets"},
+			},
+
+			"peer_subnets": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:      true,
+				ConflictsWith: []string{"peer_subnet"},
+			},
+
+			"v_net": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func AzureresourceVpcPeeringCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
You should name this properly, we use camelcase with the first letter lowercase for Golang variables/function names https://en.wikipedia.org/wiki/Camel_case